docs/summary.md

```diff
@@ -26,8 +20,16 @@ execute(link, operation).subscribe({
   error: error => console.log(`received error ${error}`),
   complete: () => console.log(`complete`),
 })
+
+//For single execution operations, a Promise can be used
+new Promise((resolve, reject) => (execute(link, operation)).subscribe(resolve, reject))
```

Comment:
@stubailo @evanshauser's had the idea to export a `toPromise(execute(link, operation))` method. Would that be enough?

We could also do `executeToPromise` but I don't like that nearly as much